internal/checker/cloudformation/rules/logging/verify_access_logging_enabled_in_api_gateway_v2.rego

```diff
@@ -0,0 +1,51 @@
+# METADATA
+# title: "Verify access logging enabled in api gateway v2"
+# description: "Verify API Gateway V2 access logging for enhanced request and response monitoring."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-stage.html
+# custom:
+#   id: CB_CFT_075
+#   severity: LOW
+package lib.cloudformation.CB_CFT_075
+import future.keywords.in
+
+resource := "AWS::ApiGatewayV2::Stage"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
```

Comment:
update logic

pass[log_settings] {
    is_valid
    some resources in input.Resources
    resources.Type == resource
    log_settings := resources.Properties.AccessLogSettings
    log_settings.DestinationArn != ""
}